umibukela/import.py

```diff
@@ -0,0 +1,21 @@
+import umibukela.models as models
```

Comment:
It's generally useful to put helper scripts into a bin/ directory at the root of the repo.